### ISSUE TITLE
What are the training details regarding MAMBA2-2.7B?

### ISSUE THREAD

**Issue by fxnie** [state-spaces/mamba#656]
Some details about mamba2-2.7b are not provided in the paper. May I ask about the nheads in mamba2-2.7b What are the parameters d_head, Training steps, Learning Rate, and how to set Num_ attention heads? If possible, please let me know. Thank you very much! It would be even better if we could provide weight files in HF format.
There is another question: how should the number of attention and MLP be set?
